tests/utils/kube.py

```diff
@@ -29,24 +29,24 @@ def _find_resource_name(resource):
 
 def get_kube_resources(kubeconfig, resource, namespace='default', name=None):
     k8s_config.load_kube_config(config_file=kubeconfig)
-    resource_name, api_name, namespaced_resource = _find_resource_name(resource)
+    resrc_name, api_name, namespaced_resource = _find_resource_name(resource)
```

Comment:
A somewhat more 'standard' shorthand for `resource` is `res` rather than `resrc`.